python/rubin/rag/extra_scripts/scrape_github.py

```diff
@@ -29,6 +29,15 @@
 from langchain.schema.document import Document
```

Comment:
this is the old way to import Document, the current way is this:
```suggestion
from langchain_core.documents.base import Document
```

Reply:
Thanks, done in [a6cc243](https://github.com/lsst-dm/rubin_rag/pull/28/commits/a6cc243c00d5cfc85d6ec0098ea4a8f8350dc361)